packages/frontend/tests/acceptance/chat/privately-with/-acceptance-test.ts

```diff
@@ -21,9 +21,32 @@ module('Acceptance | Chat | Privately With', function(hooks) {
   setupApplicationTest(hooks);
   clearLocalStorage(hooks);
 
+  module('is not logged in', function(hooks) {
+    hooks.beforeEach(async function() {
+      await visit('/chat/privately-with');
+    });
+
+    test('document.title is unchanged', async function(assert) {
```

Comment:
async is not needed here 
(I need to figure out what linter this is that will warn of that (no async when there is no await))